packages/nodes-base/credentials/AdaloApi.credentials.ts

```diff
@@ -0,0 +1,24 @@
+import {
+	ICredentialType,
+	NodePropertyTypes,
+} from 'n8n-workflow';
+
+export class AdaloApi implements ICredentialType {
+	name = 'adaloApi';
+	displayName = 'Adalo API';
+	documentationUrl = 'adalo';
+	properties = [
+		{
+			displayName: 'API Key',
+			name: 'apiKey',
+			type: 'string' as NodePropertyTypes,
+			default: '',
+		},
+		{
+			displayName: 'Application ID',
+			name: 'appId',
+			type: 'string' as NodePropertyTypes,
+			default: '',
+		},
+	];
+}
```

Comment:
Missing credential verification. You can check ServiceNow, Mattermost, and Pipedrive for an example of how to do it.

Reply:
I don't know how to check credentials for Adalo. They only expose endpoints for specific collections. Any ideas?